src/protocol/fluvio-protocol-core/src/encoder.rs

```diff
@@ -33,6 +33,20 @@ pub trait Encoder {
         buf.put_slice(&out);
         Ok(buf.freeze())
     }
+
+    /// Return a [`bytes::Bytes`] representation of the object that follows the
+    /// [Kafka Protocol](https://kafka.apache.org/protocol#The_Messages_ListOffsets).
+    /// First 4 bytes are size of the message.  Then total buffer = 4 + message content
+    fn as_kafka_frame_bytes(&self, version: Version) -> Result<Bytes, Error> {
```

Comment:
Maybe as_kafka_message is a better name?

Reply:
we are deprecating `kafka`.  change to `fluvio_frame_bytes`.   don't think this should belong here but in the codec